server-phoenix/lib/helios_web/clients/tweet.ex

```diff
@@ -20,10 +22,15 @@ defmodule HeliosWeb.Clients.Tweet do
       interactions: interactions(tweet),
       status: status(tweet),
       user: user(tweet),
-      created_at: tweet.created_at
+      created_at: created_at(tweet)
```

Comment:
Does this pass in the unit tests? Are we running the unit tests in CI?

Reply:
I completely forgot about updating unit tests! I updated them now, so we're passing them. I also added them in CI.